docs/COMPONENT_GUIDELINES.md

```diff
@@ -29,3 +29,80 @@ So solutions like `@media-queries` might not work, and `window` shouldn't be dir
 
 * `wix-ui-tpa` are rendered for Wix's users-of-users, and so they must comply with web accessibility
  principles.
+ 
+ ## Typography
+ When creating a new component that renders text, we should in most cases use the `<Text/>` component with the correct props and style.  
+ In some cases we might need to use a typography that isn't defined in the `<Text/>` component.  
+ In this case, we rely on the style processor's syntax, to ensure we get the correct font from the viewer.
```

Comment:
Do you mean here for cases that you wish to connect the text to the user's choice? if so, then I'd simply write:
```suggestion
If a specific text needs to be configurable (e.g. by a user, or for a different variation), you are advised to use the `style-processor` syntax for customing fonts (see example below)
```

And then add a simple example of how to extend the component

Reply:
Not exactly what I meant